fdbserver/Knobs.cpp

```diff
@@ -509,6 +509,7 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( BEHIND_CHECK_COUNT,                                      2 );
 	init( BEHIND_CHECK_VERSIONS,             5 * VERSIONS_PER_SECOND );
 	init( WAIT_METRICS_WRONG_SHARD_CHANCE,   isSimulated ? 1.0 : 0.1 );
+	init( DD_METRICS_REPORT_INTERVAL,                            5.0 );
```

Comment:
5s will be too frequent and will take too much space in tracing system, say Splunk. Let's start with longer interval, say 30s or 60s.

Reply:
In some discussion with @sears , we decided that this should be "pulled" rather than "pushed". We will use some other strategies.